xds/internal/xdsclient/transport/transport.go

```diff
@@ -105,7 +106,14 @@ type Transport struct {
 // cause the transport layer to send an ACK to the management server. A non-nil
 // error is returned from this function when the data model layer believes
 // otherwise, and this will cause the transport layer to send a NACK.
-type UpdateHandlerFunc func(update ResourceUpdate) error
+type OnRecvHandlerFunc func(update ResourceUpdate) error
+
+// OnSendHandlerFunc is the implementation at the xDS data model layer, which
+// handles state changes for the resource watch and stop watch timers accordingly.
+//
+// A nil error is returned from this function when the data model layer has applied
+// accurate state transitions for all the resources in the update.
```

Comment:
There is no error returned by this function. Delete this comment?

Reply:
oops. nixing it